Scheduled Tests Failed

https://github.com/hmasdev/autogen-werewolf/actions/runs/10950816939

```
_______________ ERROR collecting tests/game_master/test_base.py ________________
ImportError while importing test module '/home/runner/work/autogen-werewolf/autogen-werewolf/tests/game_master/test_base.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/game_master/test_base.py:1: in <module>
    import autogen
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/autogen/__init__.py:9: in <module>
    from .agentchat import *
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/autogen/agentchat/__init__.py:8: in <module>
    from .assistant_agent import AssistantAgent
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/autogen/agentchat/assistant_agent.py:11: in <module>
    from .conversable_agent import ConversableAgent
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/autogen/agentchat/conversable_agent.py:25: in <module>
    from ..code_utils import (
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/autogen/code_utils.py:[23](https://github.com/hmasdev/autogen-werewolf/actions/runs/10950816939/job/30406764009#step:5:24): in <module>
    from autogen import oai
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/autogen/oai/__init__.py:8: in <module>
    from autogen.oai.client import ModelClient, OpenAIWrapper
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/autogen/oai/client.py:15: in <module>
    from flaml.automl.logger import logger_formatter
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/flaml/automl/__init__.py:1: in <module>
    from flaml.automl.automl import AutoML, size
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/flaml/automl/automl.py:20: in <module>
    from flaml.automl.ml import huggingface_metric_to_mode, sklearn_metric_name_set, spark_metric_name_dict, train_estimator
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/flaml/automl/ml.py:12: in <module>
    from flaml.automl.model import BaseEstimator, TransformersEstimator
/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/flaml/automl/model.py:19: in <module>
    import sklearn
E   ModuleNotFoundError: No module named 'sklearn'
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,6 +31,7 @@ dev = [
     "pytest-cov>=5.0.0",
     "pytest-mock>=3.14.0",
     "jinja2>=3.1.4",
+    "flaml!=2.3.0",        # patch for #28
 ]
 
 [tool.uv]

